tactical-microgrid-standard/CMakeLists.txt

```diff
@@ -5,23 +5,24 @@ enable_testing()
 
 find_package(OpenDDS REQUIRED)
 
-add_library(Common
+add_library(OpenDDS_TMS
```

Comment:
Should the common library be named something like TMS_Common? I think that would be more specific to the functionality of this library.